docs/settings/apm-settings.asciidoc

```diff
@@ -74,6 +74,9 @@ Changing these settings may disable features of the APM App.
 | `apm_oss.sourcemapIndices`
   | Matcher for all {apm-server-ref}/sourcemap-indices.html[source map indices]. Defaults to `apm-*`.
 
+| New setting
+  | New setting description
```

Comment:
What is the name of the config variable for changing the default `observability-annotations` index? Even better, where would I be able to find that in https://github.com/elastic/kibana/pull/64796?

Reply:
it's `xpack.observability.annotations.index`: https://github.com/elastic/kibana/pull/64796/files#diff-903c2c668d8d0f9d4904c46147ef23f0R13